README.md

```diff
@@ -13,6 +13,48 @@ The Structural Simulation Toolkit (SST) was developed to explore innovations in
 
 ---
 
+## Install
+
+### Install with Spack
+
+```
+spack install sst-elements
+```
+
+### (TODO) Pull the docker container
+
+```
+docker pull sst-docker-image-name:latest-tag
+```
+
+## Building
```

Comment:
Need build from git clone and build from tarball separately.  Also I vote we pick 2 operating systems and give a complete build. I like it when the README shows all the required packages.